### PR TITLE
fix: preserve imports when parsing to json and then stringifying back to proto

### LINKED ISSUE #62: Add imports to stringify method
I'd like to be able to use the library as part of a protobuf file generation project I am working on but am running into an issue where imports are not preserved when calling `parse` to transform protobuf to JSON and then calling `stringify` to stringify the JSON back into the `.proto` format.

It seems like a solution to this would be pretty simple. Happy to create a PR if desired.

### PR DESCRIPTION
**Pull request by kyle-schuetz**
Fixes #62

### CODE REVIEW

File: test/fixtures/import.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+					"oneof": null,
+					"required": false,
+					"repeated": true,
+					"options": {}
+				}
+			]
+		}
+	],
+	"options":{}
+}
```
Comment (mafintosh) [inline test/fixtures/import.json:30]: can you indent this by 2 spaces instead of a tab to make it consistent with the other fixtures?

**Comment by mafintosh** [comment]
Looking really good other that my whitespace comment, thanks a lot!

File: test/fixtures/import.json (hunk 1 above)
Comment (kyle-schuetz) [reply test/fixtures/import.json:30]: Definitely can do that. I got lazy and didn't set an editor config when making my changes :)

**Comment by mafintosh** [comment]
@kyle-schuetz good to go yes?

File: test/fixtures/import.json (hunk 1 above)
Comment (kyle-schuetz) [reply test/fixtures/import.json:30]: OK, think I've got it set up with two spaces rather than a tab.

Comment (mafintosh) [reply test/fixtures/import.json:30]: Perfect

**Comment by kyle-schuetz** [comment]
I am good to go if you are @mafintosh

**Comment by mafintosh** [comment]
woot, 3.6.0

**Pull request merged**